lstchain/reco/r0_to_dl1.py

```diff
@@ -369,7 +392,19 @@ def r0_to_dl1(
             # a candidate muon ring. In that case the full image could be kept, or reduced
             # only after the ring analysis is complete.
 
-            for ii, telescope_id in enumerate(event.dl1.tel.keys()):
+            for telescope_id, dl1_tel in event.dl1.tel.items():
+                dl1_tel.prefix = ''  # don't really need one
```

Comment:
what is the inizialization of `dl1_tel.prefix`?

Reply:
`dl1_camera` created from the name of the container class :`DL1CameraContainer`. But this is not a change, the code is just at a new position.